libs/dh/message-archive/feature-log-search/src/lib/blobcontent/dh-message-archive-log-search-blob-content.component.scss

```diff
@@ -0,0 +1,16 @@
+/**
```

Comment:
Suggestion: remove this file since it's empty. A component doesn't have to include any styles, just remove the `styleUrls` from `libs\dh\message-archive\feature-log-search\src\lib\blobcontent\dh-message-archive-log-search-blob-content.component.ts`